dev/com.ibm.ws.app.manager.war/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -37,3 +37,6 @@ earapp.addWebModuleClassPathTo.name=Web module class path loader
 earapp.addWebModuleClassPathTo.desc=This attribute defines the class loader that is used for JARs that are referenced by web application Class-Path headers.
 earapp.addWebModuleClassPathTo.warLoader=Use the web application class loader for the JARs that are referenced by a web application Class-Path header.
 earapp.addWebModuleClassPathTo.earLoader=Use the parent enterprise application class loader for the JARs that are referenced by a web application Class-Path header.
+
+earapp.annotationScanLibaray.name=Annotation scan libraries
+earapp.annotationScanLibaray.desc=Configure which libraries are going to be scanned for Web Application annotations in a comma seperated list. Valid options are earLibrary, manifestClassPath, or ALL.
```

Comment:
Should `ALL` be all caps or just `all`?

Reply:
`Library` is spelled wrong here as `Libaray` and in the metatype.xml file as well.